lib/generator.ml

```diff
@@ -272,9 +268,10 @@ let make_upgraders ?(impl_kind = Config.Native) ?output_prefix = function
           Error e)
     in
     let* versions = get_versions [] files in
-    let file_versions = List.sort ~cmp:Int.compare versions in
+    (* TODO: write a custom ordering function, where "2" < "10" *)
+    let file_versions = List.sort ~cmp:String.compare versions in
```

Comment:
I think the ordering of versions should be based on the order of files passed to stork.

Reply:
it especially makes sense with strings and with #12

Reply:
that was my other thought, except then you can't use `file_*.atd` wildcards, and have to pass the versions explicitly

Reply:
I think it's fine. We don't want people to pass files (#12) anyway. The wildcards don't make sense in that scenario.